tests/js/server/utils-tests.js

```diff
@@ -0,0 +1,126 @@
+/*
+
+  Tests for the gpii.express.user.utils component.
+
+ */
+/* eslint-env node */
+"use strict";
+
+var fluid        = require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+
+require("../../../");
+require("../lib/");
+
+var jqUnit       = require("node-jqunit");
+
+fluid.registerNamespace("gpii.tests.express.user.utils.caseHolder");
+
+gpii.tests.express.user.utils.createUser = function (utils) {
+    var prom = utils.createNewUser({
+        username: "myFirstUser",
+        password: "this is a password",
+        email: "myFirstUser@gpii.net"
+    });
+    prom.then(function (data) {
+        jqUnit.assertEquals("Generated Couch ID", "org.couch.db.user:myFirstUser", data._id);
+        jqUnit.assertEquals("Email", "myFirstUser@gpii.net", data.email);
+        jqUnit.assertEquals("Username", "myFirstUser", data.username);
+        jqUnit.assertTrue("Unlock the password", gpii.express.user.utils.verifyPassword(data, "this is a password"));
+    }, function (err) {
+        jqUnit.fail("Unable to create user with error: " + err);
+    });
+    return prom;
+};
+
+gpii.tests.express.user.utils.unlockUsernamePassword = function (utils) {
```

Comment:
As far as I can see this function can also be eliminated in favour of better `reject` and `resolve` entries in your test sequence step.

Reply:
this makes sense and is done